### ISSUE TITLE
Unable to `npm install` phaser template due to `package-lock` pointing to apple npm registry

### ISSUE THREAD

**Issue by edmundito** [SpringRoll/Springroll-Seed#95]
I'm trying to resolve an issue where setting up and installing from the Phaser template gives me the following error:

```
npm ERR! network request to https://artifacts.apple.com/api/npm/npm-private/process/-/process-0.11.10.tgz failed, reason: getaddrinfo ENOTFOUND artifacts.apple.com
```

The changes with the Apple registry were introduced when migrating to Webpack 5: https://github.com/SpringRoll/Springroll-Seed/commit/9ae0063437d85f7910d64372acdd93bd82e25ebd

An easy workaround is to update the address in `package-lock` from the Apple registry to npmjs.org.

**Comment by 902seanryan** [comment]
nice catch @edmundito I'll be fixing this in a 1.3.1 release soon, appreciate the work

**Comment by 902seanryan** [comment]
@edmundito this should be corrected in the 1.3.1 releases. I'm going to close the issue but if, somehow, it persists feel free to re-open
